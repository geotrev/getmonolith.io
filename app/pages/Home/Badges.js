import React from "react"
import Markdown from "react-markdown"
import statusBadges from "./markdown/badges.md"

const Badges = () => <Markdown source={statusBadges} escapeHtml={false} />

export default Badges
