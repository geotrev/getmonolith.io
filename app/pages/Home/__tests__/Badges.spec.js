import React from "react"
import Badges from "../Badges"

jest.mock(
  "../markdown/badges.md",
  () => "some status content, converted from markdown to html/text!"
)

describe("<Badges />", () => {
  it("renders", () => {
    const wrapper = mount(<Badges />)
    expect(wrapper).toMatchSnapshot()
  })
})
