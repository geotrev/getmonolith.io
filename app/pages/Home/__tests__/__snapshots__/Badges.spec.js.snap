// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badges /> renders 1`] = `
<Badges>
  <ReactMarkdown
    astPlugins={Array []}
    escapeHtml={false}
    parserOptions={Object {}}
    plugins={Array []}
    rawSourcePos={false}
    renderers={Object {}}
    skipHtml={false}
    source="some status content, converted from markdown to html/text!"
    sourcePos={false}
    transformLinkUri={[Function]}
  >
    <Root
      key="root-1-1"
    >
      <p
        key="paragraph-1-1"
      >
        <TextRenderer
          key="text-1-1"
          nodeKey="text-1-1"
          value="some status content, converted from markdown to html/text!"
        >
          some status content, converted from markdown to html/text!
        </TextRenderer>
      </p>
    </Root>
  </ReactMarkdown>
</Badges>
`;
